test/manual/templates/system_api_legacy_tests.sh

```
@@ -0,0 +1,89 @@
+# tests/map_tests_legacy.sh
+#!/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)